SteamKit2/SteamKit2/Steam/Handlers/SteamUnifiedMessages/Callbacks.cs

```diff
@@ -3,114 +3,33 @@
  * file 'license.txt', which is part of this source code package.
  */
 
-using System;
-using System.Reflection;
 using ProtoBuf;
 
 namespace SteamKit2
 {
-    public partial class SteamUnifiedMessages : ClientMsgHandler
+    public partial class SteamUnifiedMessages
     {
         /// <summary>
         /// This callback is returned in response to a service method sent through <see cref="SteamUnifiedMessages"/>.
         /// </summary>
-        public class ServiceMethodResponse : CallbackMsg
+        public class ServiceMsg<TResult> : CallbackMsg where TResult : IExtensible, new()
```

Comment:
```suggestion
        public class ServiceMethodResponse<TResult> : CallbackMsg where TResult : IExtensible, new()
```

Let's keep the same class name.